_pages/home.md

```diff
@@ -22,30 +22,67 @@ scientific Python packages. We also build technical capacity by providing a
-3. Volunteer to be a reviewer for pyOpenSci. We are looking for people who can help review software packages as they are submitted.
-4. Spread the word! Please spread the word about this effort. And look for us this summer at the SciPy meeting in Austin, Texas!
-
-## Who's Involved
-
-pyOpenSci is a new effort being lead by a small team of Python enthusiasts including:
-
-* Leah Wasser, Director of Earth Analytics Education -- Earth Lab, University of Colorado -- Boulder.
-* Max Joseph, Data Scientist -- Earth Lab, University of Colorado -- Boulder.
-* Jenny Palomino, Faculty and Curriculum Developer -- Earth Lab, University of Colorado -- Boulder.
-* Chris Holdgraf, Project Jupyter, Berkeley - BIDS.
-* Luiz Irber, DIB Lab -- UC Davis
-* Paige Bailey, Google
-* Kylen Solvik, Graduate Student -- Earth Lab, University of Colorado -- Boulder.
-* Carson Farmer, External advisor -- Textile.io.
-* Leonardo Uieda, Visiting Researcher -- University of Hawai'i at Mānoa -- Honolulu.
-* Filipe Fernandes, IOOS
+1. **[Join a community meeting](#community-meetings)**: We have monthly video meetings that you are welcome to attend.
+2. **[Join our community forum](https://pyopensci.discourse.group/)**. This is an online space
+   for discussion within the pyOpenSci community. Announcements for new meetings will be posted here.
+3. **Submit a package to pyOpenSci**. We are looking for Python packages that
+   support scientific Python community. Read more about our
+   [review process and scope](https://www.pyopensci.org/dev_guide/peer_review/aims_scope.html)
+4. **Volunteer to be a reviewer for pyOpenSci**. We are looking for people who
+   can help review software packages as they are submitted.
+5. **Help with technical infrastructure**. pyOpenSci has a few pieces of technical
+   infrastructure (like this site) which need maintenance and development.
+6. **Spread the word**! pyOpenSci is a community-driven project. Tell your friends,
+   recommend that a colleague submit a package, mention us at conferences, spread the word!
+
+### Community meetings
+
+Currently, the pyOpenSci team holds community meetings roughly every 2 weeks. These are
+held remotely, and open to anyone who would like to attend. We discuss ideas for improving
+the process / technology / community management / etc of the pyOpenSci community, and
+would value your participation!
+
+The best place to find information about the next community meeting is
+[in the community forum](https://pyopensci.discourse.group). We post dates / times
+for new meetings there.
+
+### Who's Involved
+
+See our [contributors page](contributors.html) for a list of the contributors
+in the pyOpenSci community.
+
+## FAQ
+
+### What's the difference between pyOpenSci and JOSS?
+
+The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
+to improving the visibility and quality of scientific software. They do
+so by providing a lightweight review and publishing process so that authors
```

Comment:
"lightweight" really minimizes the JOSS review process. I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation. It would be more fair for pyopensci to simply state the differences in the review process and let the reader decide whether one happens to be lightweight or not.

Reply:
> "lightweight" really minimizes the JOSS review process. I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation. It would be more fair for pyopensci to simply state the differences in the review process and let the reader decide whether one happens to be lightweight or not.

Yeah, that's a fair point. I guess I was trying to say _lighter_ weight than pyOpenSci but I see what you mean @moorepants about potentially framing the JOSS review process in a negative light using language like this.

> I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation.

💯absolutely.

Reply:
Thanks for noticing this - I agree we definitely don't want to convey the idea that JOSS is a less-than-rigorous review process. Could somebody suggest some better wording that conveys the ideas we want?

Reply:
I would remove "lightweight" as a start.

Reply:
https://github.com/pyOpenSci/pyopensci.github.io/pull/27 there's a start - feel free to suggest edits and I'm happy to iterate there

Reply:
thanks guys!! we want everyone to be happy with how we represent our partnership on the website. all feedback is very much appreciated!!!